community/xbps/APKBUILD

```diff
@@ -32,13 +26,7 @@ build() {
 }
 
 package() {
-	cd "$builddir"
-
 	make DESTDIR="$pkgdir/" install
-
-	install -D -m 644 COPYING "$pkgdir/usr/share/licenses/$pkgname/COPYING"
-	install -D -m 644 COPYING.3RDPARTY \
```

Comment:
This was renamed to LICENSE.3RDPARTY, but is still relevant, as we're distributing the software in binary form.
However, all of them could be added into `license=` separately, I guess.
From what I'm seeing, there is (in parenthesis are SPDX identifiers):

- University of California (BSD-3-Clause)
- Alistair G. Crooks (BSD-3-Clause)
- Dag-Erling Coïdan Smørgrav (BSD-3-Clause)
- Todd C. Miller (ISC)
- The NetBSD Foundation (BSD-2-Clause-NetBSD)

Please do double-check!